.github/workflows/ci.yml

```diff
@@ -1,6 +1,16 @@
 name: ci
 
-on: [push, pull_request]
+on:
+  push:
+    paths-ignore:
+      - docs/**
+      - examples/**
```

Comment:
Examples are linted.

Reply:
Wouldn't it be better to separate the lint workflow in another file? Because having the whole matrix running on just change on examples isn't really necessary.

Reply:
Not sure how often that happens but yes this would make sense.

Reply:
I'll setup this in another PR.

Reply:
But then don't forget to re-add `examples`.

Reply:
Looking at the git history of `examples` it looks useless complexity addition to me. Keeping the `paths-ignore` like this is ok.